udata/sentry.py

```diff
@@ -56,10 +62,24 @@ def init_app(app):
         for exception in IGNORED_EXCEPTIONS:
             exceptions.add(exception)
 
+        sentry_sample_rate: float = 0.1
+        if app.config.get('DEBUG', None):
+            sentry_sample_rate = 1.0
+
         sentry_sdk.init(
             dsn=app.config['SENTRY_PUBLIC_DSN'],
             integrations=[FlaskIntegration(), CeleryIntegration()],
-            ignore_errors=list(exceptions)
+            ignore_errors=list(exceptions),
+            release=f"udata@{package_version('udata')}",
+            environment=app.config.get('SITE_ID', None),
+            # Set traces_sample_rate to 1.0 to capture 100%
+            # of transactions for performance monitoring.
+            # Sentry recommends adjusting this value in production.
+            traces_sample_rate=sentry_sample_rate,
+            # Experimental profiling
+            _experiments={
+                'profiles_sample_rate': sentry_sample_rate,
+            },
```

Comment:
I would say that we should log 100% of errors by default for now (and make it configurable) but only a portion of profiling indeed. Ignore this comment if this is what it does ;)

Reply:
Yes after some inquiry, this is only for perf so let's keep as it is.

Reply:
But: Sentry sample rate has been changed as an app config.